test/demo/index.php

```diff
@@ -9,7 +9,7 @@
 
 ini_set('session.save_handler', 'files');
 
-$autoload = __DIR__ . '/../../vendor/autoload.php';
+$autoload = '../../../../autoload.php';
```

Comment:
As of PHP >= 7.0 this could be resolved using `dirname(__DIR__, 2)` because `__DIR__ . '/../../' returns the path to demo/index.php plus the string '/../../' as I have already observed in several machines running XAMPP.

I replaced __DIR__ with the relative path to composer's autoload to save backwards compatibility with older php versions while providing usability with >7.0

Reply:
Thought I commented the fix in the request description but I must have forgotten it. Sorry

Reply:
`__DIR__` is supported since PHP 5.3.0. I double tested the demo script to be sure, everything works as expected. Maybe it's an issue of your environment using XAMPP.